src/Jackett.Common/Definitions/thepiratebay.yml

```
@@ -162,6 +162,9 @@ search:
       selector: category
     title:
       selector: name
+      filters:
+        - name: re_replace
+          args: ["( - )(?!.* - )", " -"]
```

```suggestion
          args: [" ?(?:- )(\\w+)$", "-$1"]
```
Includes:
`Title 2023- YIFY`
`Title - 2023 - YIFY`

Excludes:
`Title - 2023-YIFY`
`Title - 2023 YIFY`

---

Edited above, though that picks up `Title - 2023-D-Zone`.